scripts/install

```diff
@@ -30,130 +31,163 @@ admin=$YNH_APP_ARG_ADMIN
 #=================================================
 # CHECK IF THE APP CAN BE INSTALLED WITH THESE ARGS
 #=================================================
+ynh_script_progression --message="Validating installation parameters..." --weight=2
 
 final_path=/var/www/$app
 test ! -e "$final_path" || ynh_die "This path already contains a folder"
 
-path_url=$(ynh_normalize_url_path $path_url)	# Check and normalize path
-
-# Check web path availability
-ynh_webpath_available $domain $path_url
 # Register (book) web path
-ynh_webpath_register $app $domain $path_url
+ynh_webpath_register --app=$app --domain=$domain --path_url=$path_url
 
 #=================================================
 # STORE SETTINGS FROM MANIFEST
 #=================================================
 
-ynh_app_setting_set $app domain "$domain"
-ynh_app_setting_set $app path_url "$path_url"
-ynh_app_setting_set $app admin "$admin"
+ynh_app_setting_set --app=$app --key=domain --value=$domain
+ynh_app_setting_set --app=$app --key=path --value=$path_url
+ynh_app_setting_set --app=$app --key=admin --value=$admin
 
 #=================================================
 # STANDARD MODIFICATIONS
 #=================================================
 # INSTALL DEPENDENCIES
 #=================================================
+ynh_script_progression --message="Installing dependencies..." --weight=12
 
-ynh_install_app_dependencies "$PKG_DEPENDENCIES"
+ynh_install_app_dependencies $pkg_dependencies
 
 #=================================================
-# CREATE A MYSQL DB
+# CREATE A MYSQL DATABASE
 #=================================================
+ynh_script_progression --message="Creating a MySQL database..."
 
 db_name=$(ynh_sanitize_dbid $app)
-db_user="$db_name"
-
-# Generate random DES key & password
-deskey=$(ynh_string_random 24)
-ynh_app_setting_set "$app" deskey "$deskey"
-ynh_app_setting_set "$app" db_name "$db_name"
+db_user=$db_name
+ynh_app_setting_set --app=$app --key=db_name --value=$db_name
+ynh_mysql_setup_db --db_user=$db_user --db_name=$db_name
 
-# Initialize database
-ynh_mysql_setup_db "$db_user" "$db_name"
 #=================================================
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
+ynh_script_progression --message="Setting up source files..." --weight=6
+
+### `ynh_setup_source` is used to install an app from a zip or tar.gz file,
+### downloaded from an upstream source, like a git repository.
+### `ynh_setup_source` use the file conf/app.src
```

Comment:
Please remove these lines

(Impossible to commit with my phone right now /o\)

Reply:
Done

Reply:
Thx